4.analyze_data/notebooks/UMAP/nbconverted/2.plate4_find_sc.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Generate single-cell crop images based on the plate 4 UMAP coordinates
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import pathlib
+import cv2
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+umap_df = pd.read_csv(pathlib.Path("./results/UMAP_localhost231120090001_sc_feature_selected.tsv.gz"), sep='\t')
+
+images_dir = pathlib.Path("../../../1.preprocessing_data/Corrected_Images/localhost231120090001/")
+
+plate = "localhost231120090001"
+
+output_dir = pathlib.Path("plate4_umap_crops")
+output_dir.mkdir(parents=True,exist_ok=True)
+
+# Output dir for composite and cropped images
+output_img_dir = pathlib.Path("./images")
+output_img_dir.mkdir(exist_ok=True)
+
+
+# ## Find single-cells per cluster
```

Comment:
consider removing vestigial code

Reply:
I am not sure the code that you are referring to, but I will make sure to remove any unused code!